contracts/SimpleSwap.sol

```diff
@@ -116,53 +97,39 @@ contract SimpleSwap is SW3Utils {
   /// @param value maximum amount to send
   /// @return payout amount that was actually paid out
   /// @return payout amount that bounced
-  function _payout(address beneficiary, uint value) internal returns (uint payout, uint bounced) {
-    /* part of hard deposit used */
-    payout = Math.min256(value, hardDeposits[beneficiary].amount);
```

Comment:
I don't see why a beneficiary would not be able to request a payout more than his hard deposit balance.